static/less/flow_editor.less

```diff
@@ -2214,4 +2249,88 @@ div.modal[role="dialog"] {
       word-wrap: break-word;
     }
   }
+}
+
+.modal {
+  .cursor-pointer {
+    cursor: pointer;
+  }
+  .cursor-pointer:hover {
+    opacity: 0.7;
```

Comment:
If you want a hover style for the quick reply button, then you should apply it to the attachment button to for consistency